cloudant-client/src/main/java/com/cloudant/client/org/lightcouch/internal/CouchDbUtil.java

```diff
@@ -127,12 +127,15 @@ public static String removeExtension(String fileName) {
     }
 
     public static String streamToString(InputStream in) {
-        Scanner s = new Scanner(in, "UTF-8");
-        s.useDelimiter("\\A");
-        String str = s.hasNext() ? s.next() : null;
-        close(in);
-        s.close();// mdb
-        return str;
+        try {
+            Scanner s = new Scanner(in, "UTF-8");
+            s.useDelimiter("\\A");
+            String str = s.hasNext() ? s.next() : null;
+            s.close();// mdb
```

Comment:
Yes, no idea what the context of mdb was. I agree it is probably slightly out of scope, but I will replace this `Scanner` code.